File: SimpleCalculator/app/src/main/java/com/mhmmdyzci/simplecalculator/MainActivity.java
package com.mhmmdyzci.simplecalculator;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    EditText numberOne;
    EditText numberTwo;
    TextView resultText;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
         numberOne = findViewById(R.id.numberOne);
         numberTwo = findViewById(R.id.numberTwo);
         resultText = findViewById(R.id.resultText);

    }
    public void sum(View view){
        if(numberOne.getText().toString().matches("") || numberTwo.getText().toString().matches(" ")){
            resultText.setText("Enter number!");
        } else {
            int number1 = Integer.parseInt(numberOne.getText().toString());
            int number2 = Integer.parseInt(numberTwo.getText().toString());
            int result = number1 + number2 ;
            resultText.setText("Result: " + result );

        }


    }
    public void deduct(View view){
        if(numberOne.getText().toString().matches("") || numberTwo.getText().toString().matches(" ")){
            resultText.setText("Enter number!");
        } else {
            int number1 = Integer.parseInt(numberOne.getText().toString());
            int number2 = Integer.parseInt(numberTwo.getText().toString());
            int result = number1 - number2 ;
            resultText.setText("Result: " + result );

        };

    }
    public void multiply(View view){
        if(numberOne.getText().toString().matches("") || numberTwo.getText().toString().matches(" ")){
            resultText.setText("Enter number!");
        } else {
            int number1 = Integer.parseInt(numberOne.getText().toString());
            int number2 = Integer.parseInt(numberTwo.getText().toString());
            int result = number1 * number2 ;
            resultText.setText("Result: " + result );

        };

    }
    public void divide(View view){
        if(numberOne.getText().toString().matches("") || numberTwo.getText().toString().matches(" ")){
            resultText.setText("Enter number!");
        } else {
            int number1 = Integer.parseInt(numberOne.getText().toString());
            int number2 = Integer.parseInt(numberTwo.getText().toString());
            int result = number1 / number2 ;
            resultText.setText("Result: " + result );

        };

    }
}

File: CalculatorKotlin/app/src/main/java/com/mhmmdyzci/calculatorkotlin/MainActivity.kt
package com.mhmmdyzci.calculatorkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.mhmmdyzci.calculatorkotlin.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding
     var number1 : Int? = null
     var number2 : Int? = null
     var result : Int? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)
    }
    fun mySum(view: View){
         number1 = binding.editText.toString().toIntOrNull()
         number2 = binding.editText2.toString().toIntOrNull()
        if (number1 == null || number2 == null){
            binding.resultText.text = "Error"

        }else{
            result = number1!! + number2!!
            binding.resultText.text = result.toString()

        }

    }

    fun mySub(view: View){
        number1 = binding.editText.toString().toIntOrNull()
        number2 = binding.editText2.toString().toIntOrNull()
        if (number1 == null || number2 == null){
            binding.resultText.text = "Error"

        }else{
            result = number1!! - number2!!
            binding.resultText.text = result.toString()

        }

    }
    fun myMultiply(view: View){
        number1 = binding.editText.toString().toIntOrNull()
        number2 = binding.editText2.toString().toIntOrNull()
        if (number1 == null || number2 == null){
            binding.resultText.text = "Error"

        }else{
            result = number1!! * number2!!
            binding.resultText.text = result.toString()

        }

    }

    fun myDiv(view: View){
        number1 = binding.editText.toString().toIntOrNull()
        number2 = binding.editText2.toString().toIntOrNull()
        if (number1 == null || number2 == null){
            binding.resultText.text = "Error"

        }else{
            result = number1!! / number2!!
            binding.resultText.text = result.toString()

        }

    }




}